hw/ip/sysrst_ctrl/data/sysrst_ctrl.hjson

```diff
@@ -594,7 +594,7 @@
       swaccess: "rw",
       hwaccess: "hro",
       regwen: "REGWEN",
-      resval: "0",
+      resval: "2000",
```

Comment:
o i was referring to this one.

Reply:
Oh yeah this one (I somehow was thinking you referred to a reset signal).
I changed it since this was spec'ed like that in the one pager document:
https://docs.google.com/document/d/1LgPRkYrpIgT-7VZhlfSrjTzWcUeVC2VKDnJ8xna655o/edit#heading=h.uwguf9669yhw